torch/csrc/api/include/torch/nn/modules/activation.h

```diff
@@ -849,7 +849,8 @@ class TORCH_API MultiheadAttentionImpl
 
   std::tuple<Tensor, Tensor> forward(const Tensor& query, const Tensor& key,
                  const Tensor& value, const Tensor& key_padding_mask = {},
-                 bool need_weights = true, const Tensor& attn_mask = {});
+                 bool need_weights = true, const Tensor& attn_mask = {},
+                 bool average_attn_weights = true);
  protected:
   FORWARD_HAS_DEFAULT_ARGS({3, AnyValue(Tensor())}, {4, AnyValue(true)}, {5, AnyValue(Tensor())})
```

Comment:
I think `average_attn_weights` (arg 6) might need to be added to this

Reply:
nice catch :)